lua/packer/display.lua

```diff
@@ -386,10 +398,15 @@ local display_mt = {
   end),
 
   --- Display the final results of an operation
-  final_results = vim.schedule_wrap(function(self, results, time)
+  -- TODO would be nice to have the option to not show all the commits but just the plugins, one per line
```

Comment:
I agree that this would be nice! (though it can be in a later PR, particularly if this feature is opt-in)

Reply:
Forgot to remove this todo and another one. This is what #1035 enabled :)

Removed the TODOs